test/linalg/spectral.jl

```diff
@@ -97,8 +97,6 @@ Matrix(nbt::Nonbacktracking) = Matrix(sparse(nbt))
             T = eltype(g)
             amat = adjacency_matrix(g, Float64; dir=dir)
             lmat = laplacian_matrix(g, Float64; dir=dir)
-            @test isa(amat, SparseMatrixCSC{Float64,T})
-            @test isa(lmat, SparseMatrixCSC{Float64,T})
```

Comment:
do we need to test `amat` anywhere?

Reply:
I don't know the purpose of testing the type of `amat`, but if we have to test it, we'll need a function similar to `_find_correct_type` since the type of colptr of adjacency matrix is no longer the same as the eltype of the graph.

Reply:
Coverage looks fine, actually.